README.md

```diff
@@ -251,10 +260,24 @@ The action supports the following inputs:
    for available range specifications). Examples are: `"<1.2.0"`, `"~1.1.0"`, `"1.1.7"` (all three installing
    the latest available `1.1` version). Prerelease versions can be specified and a range will stay within the
    given tag such as `beta` or `rc`. If no version is given, it will default to `latest`.
+- `terraform_version_file` - (optional) The path to a file containing terraform version. Supported file types are `.tool-versions` or anything else. See more details in [about version-file](#Terraform-version-file).
 - `terraform_wrapper` - (optional) Whether to install a wrapper to wrap subsequent calls of
    the `terraform` binary and expose its STDOUT, STDERR, and exit code as outputs
    named `stdout`, `stderr`, and `exitcode` respectively. Defaults to `true`.
 
+### Terraform version file
+
+If the `terraform_version_file` input is specified, the action will extract the version from the file and install it.
+
+Supported files names are `.tool-versions` or anything else.
```

Comment:
We use `.terraform-version` which is a quasi-standard for Terraform: https://github.com/tfutils/tfenv?tab=readme-ov-file#terraform-version-file.

Reply:
Do you mean replace `or anything else.` for `.terraform-version`?

I think keeping compatibility with https://github.com/asdf-vm/asdf by using `.tool-versions` is a good idea, for instance, some other GitHub actions use it:
- https://github.com/updatecli/updatecli-action/#updatecli-version-file
- https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#Java-version-file

Reply:
The Java action also supports .java-version: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md.

I think it would be good to mention .terraform-version specifically because a lot of people are probably using it.

Reply:
Sure, see [`149638b` (#454)](https://github.com/hashicorp/setup-terraform/pull/454/commits/149638b3721b1540d7d6eef0d0eff5f2da2b1f79)